README.md

```diff
@@ -74,13 +74,12 @@ Automatically creates a scope (default: `examples`) and stream (default: `turbin
 #### Running
 Run the sample from the command-line:
 ```
-$ bin/run-example.sh [--controller <URI>] [--scope <name>] [--stream <name>]
-                     [--startTime <long>] [--output <path>]
+$ bin/run-example [--controller <URI>] [--input <name>] [--startTime <long>] [--output <path>]
```

Comment:
I think that `--input <name>` could be misleading if it's a `StreamId`. Perhaps this instead?
```
[--input <scope>/<stream>]
```